sorc/emcsfc_snow2mdl.fd/model_grid.f

```diff
@@ -575,33 +572,23 @@ subroutine read_mdl_grid_info
  deallocate (lats_mdl_temp, lons_mdl_temp)
 
  return
+ end
 
- end subroutine read_mdl_grid_info
```

Comment:
Did you remove this line on purpose?

Reply:
No.   Accident.   Compile failure identified the error but I thought I fixed it before committing and pushing.

added @note in place of remarks, and committed